readme.md

```diff
@@ -76,20 +76,36 @@ The codebase also has a number of reference examples.  These are all marked with
 ```
 
 
-## Descriptions of Optional Features
+## Optional Features List
+```
+bootstrap_messages
+crispy_forms
+debug_toolbar
+django_react
+django_ses
+django_social
+django_storages
+docker
+elastic_beanstalk
+sentry
+pre-commit
+security_settings
+sentry
+user_action_tracking
+```
```

Comment:
:art: I originally removed the list because it's very easy to have this list be out of sync with the headers. And Github already has a table of contents feature: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/